src/webgl/p5.Texture.js

```diff
@@ -510,12 +510,64 @@ export function checkWebGLCapabilities({ GL, webglVersion }) {
     : gl.getExtension('OES_texture_half_float');
   const supportsHalfFloatLinear = supportsHalfFloat &&
     gl.getExtension('OES_texture_half_float_linear');
+  const supportsCubemap = gl.getExtension('OES_texture_cube_map');
```

Comment:
I think this might only be necessary for WebGL 1, is that something you can double check? If it's built into WebGL2, then this line would be `supportsCubemap = webglVersion === constants.WEBGL2 || gl.getExtesion('OES_texture_cube_map')`

Reply:
yes , You are absolutely correct .
`WebGL2 supports cubemap textures natively, so you don't need to explicitly check for the OES_texture_cube_map extension in most cases.`
Mybad!
I'll just update this one too.